FastSimulation/TrackingRecHitProducer/python/SiTrackerGaussianSmearingRecHitConverter_cfi.py

```diff
@@ -1,8 +1,5 @@
 import FWCore.ParameterSet.Config as cms
 
-
```

Comment:
get rid of this file and all remnants of
SiTrackerGaussianSmearingRecHitConverter